app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, foreign_key: 'post_id'
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the Rails Active Callbacks `after_save` to call the `update_likes_counter` method every time a new post has been saved to the database, then make the `update_likes_counter` method private